tool/tctl/common/resource_command.go

```diff
@@ -973,6 +998,21 @@ func (rc *ResourceCommand) getCollection(client auth.ClientI) (ResourceCollectio
 			return nil, trace.Wrap(err)
 		}
 		return &lockCollection{locks: []types.Lock{lock}}, nil
+	case types.KindDatabaseServer:
+		servers, err := client.GetDatabaseServers(ctx, rc.namespace)
+		if err != nil {
+			return nil, trace.Wrap(err)
+		}
+		if rc.ref.Name == "" {
+			return &dbCollection{servers: servers}, nil
+		}
+
+		for _, server := range servers {
+			if server.GetName() == rc.ref.Name {
+				return &dbCollection{servers: []types.DatabaseServer{server}}, nil
```

Comment:
Didn't you return all matching database servers before? I think it made sense. It might be useful e.g. to troubleshoot which are registered.

Reply:
changed.